docs/development.md

```diff
@@ -11,98 +11,126 @@ See [the pull request template](../.github/pull_request_template.md) for steps t
   * [Environment Variables](#environment-variables)
   * [Docker](#docker)
   * [Local Development](#local-development)
-* [Django setup](#django-setup)
+* [Development in principle](#development-in-principle)
 * [Python code quality tooling](#python-code-quality-tooling) 
 * [Frontend code quality tooling](#frontend-code-quality-tooling) 
 
 ## Tools
 
+* [Python](https://www.python.org/)
+  * Target python version is defined in [../backend/runtime.txt](../backend/runtime.txt).
 * [Docker](https://docker.com)
+* [Node.js](https://nodejs.org/en/download/package-manager) for end-to-end testing with Cypress.
+* [CF8](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) for access to the cloud.gov CLI.
 * Local dev
   * [Pyenv](https://github.com/pyenv) for managing Python versions
   * [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) for managing virtual environments
   * [Postgres](https://www.postgresql.org/)
   * [SAM.gov](https://sam.gov/content/home) to validate UEI's
 
+---
+**NOTE** - Windows users *may* need the following tools installed in addition to the above tools.
+* [Chocolatey](https://chocolatey.org/install) - a package manager that will allow for installation of tools that are not available on Windows, such as GNU make (used for running Makefile operations).
+  * Here is an example of how you would use Chocolatey in the terminal to install GNU make:
+    ```
+    choco install make
+    ```
+---
+
 ## Setting up your dev environment
 
----
-**NOTE**
+### EditorConfig
 
-Target python version is defined in [../backend/runtime.txt](../backend/runtime.txt)
+We have a `.editorconfig` file at the root directory with basic settings.
+See [editorconfig.org](https://editorconfig.org/) for more information.
 
 ---
 
-## EditorConfig
+**NOTE** - For Windows developers
 
-We have a `.editorconfig` file at the root directory with basic settings.
-See [editorconfig.org](https://editorconfig.org/) for more information.
+By default, your IDE will acknowledge `CRLF` as the denotation for newlines. Many scripts in this repository **require** `LF` denotation (noticeably the .sh files when you first try to run and test the application). Make sure your IDE uses `LF` denotation for newlines when reading through each of these scripts.
+
+You can use the Git commands below BEFORE cloning the repository to your local computer to prevent files from using `CRLF` by default.
+```
+# don't use --global if you only want this behavior for a specific repository.
+git config --global core.autocrlf false
+# this ensures all files use LF.
+git config --global core.eol lf
+```
+---
 
-## Environment Variables
+### Environment Variables
 
 Create a .env file in the `/backend` directory.
 Add and define the following environment variables using the instructions below.
 
 ```
 ENV = 'LOCAL'
-SAM_API_KEY =
-SECRET_KEY =
-DJANGO_SECRET_LOGIN_KEY =
-LOGIN_CLIENT_ID =
-DISABLE_AUTH = 
+DISABLE_AUTH = False
+SECRET_KEY = YourSecretKey
```

Comment:
It's a little confusing the instructions say to "Generate your own random secret key" but then this string is definitely not randomly generated. Thoughts?

Reply:
Hmm... I had a similar thought after using your .env to get things running about whether I needed to actually generate some long random string or not.

Perhaps since it is just for local development we don't need to emphasize randomness? Like so:

> Create your own secret key for local development. Django uses this to provide cryptographic signing.